src/main/kotlin/net/irisshaders/lilybot/events/MessageEvents.kt

```diff
@@ -21,6 +51,8 @@ import kotlin.time.ExperimentalTime
 class MessageEvents : Extension() {
     override val name = "messageevents"
 
+    private val LOG_FILE_EXTENSIONS = setOf("log", "gz")
```

Comment:
```suggestion
    private val LOG_FILE_EXTENSIONS = setOf("log, "gz", "txt")
```

Reply:
I don't really want to treat txts as logs because its kinda annoying when lily keeps uploading non log files

Reply:
It's more useful than you think, most people upload their logs as text files

Reply:
Added